Sources/TSPL/TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -3266,7 +3266,7 @@ extension Array: Loggable where Element: MarkedLoggable { }
 }
 
 @Comment {
-  - test: `types-cant-have-multiple-implict-conformances`
+  - test: `types-cant-have-multiple-implicit-conformances`
```

Comment:
Confirmed that there isn't another test with this name, either spelled correctly or misspelled.

(If there were, we'd need to make sure joining the code listings was correct and that the tests still passed.)